channeldb/channel.go

```diff
@@ -2422,9 +2422,7 @@ func (c *OpenChannel) AdvanceCommitChainTail(fwdPkg *FwdPkg,
 
 		// With the commitment pointer swapped, we can now add the
 		// revoked (prior) state to the revocation log.
-		//
-		// TODO(roasbeef): store less
```

Comment:
Yay for finally resolving a [5 year old TODO](https://github.com/lightningnetwork/lnd/commit/374cab7467dd95abb9b5dc6b4e63df1fcce9bcae)!